test/test_torch.py

```diff
@@ -2397,29 +2397,23 @@ def test_tensor_factory(self):
             self.assertEqual(5., res1[0].item())
 
     def test_tensor_factory_copy_var(self):
```

Comment:
Do the changes here test the bug introduced in #11061 ? I don't see any `as_tensor` calls.

Reply:
No it doesn't, yeah I should add tests for `as_tensor` as well